tests/tck-build-logic/src/main/groovy/org.graalvm.internal.tck-harness.gradle

```diff
@@ -90,7 +90,7 @@ if (project.hasProperty("baseCommit")) {
 }
 
 def matrixDefault = [
-        "version": ["17", "dev"],
+        "version": ["17"],
```

Comment:
Oh, maybe just comment it out, so we know how to turn it back on.

Reply:
It is a trivial change, but okay, we can comment it (instead of deleting it) as well.

Reply:
It's trivial for you and me, but maybe not for others 😉